src/EnergyPlus/RuntimeLanguageProcessor.cc

```diff
@@ -977,7 +980,7 @@ namespace RuntimeLanguageProcessor {
 					// KeywordELSE  -- kind of a kludge
 					ReturnValue.Type = ValueNumber;
 					ReturnValue.Number = 1.0;
-					WriteTrace( StackNum, InstructionNum, ReturnValue );
+					WriteTrace( StackNum, InstructionNum, ReturnValue,seriousErrorFound );
```

Comment:
There are spots in here where there is no space after the comma, I'll clean those up as I push this to merge.